src/Generator/Options.cs

```diff
@@ -110,6 +110,11 @@ public Module AddModule(string libraryName)
         /// </summary>
         public bool GenerateDebugOutput;
 
+        /// <summary>
+        /// If set to true, the generated code will always be formatted.
+        /// </summary>
+        public bool GenerateFormatted;
```

Comment:
Please drop this option and always format the code instead. I.e. remove GenerateUnformatted.

Reply:
This is even better, will do